kof-operator/internal/controller/clusterdeployment_controller_test.go

```diff
@@ -460,5 +461,57 @@ var _ = Describe("ClusterDeployment Controller", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(configMapCDGeneration).To(BeNumerically(">", initialClusterDeploymentGeneration))
 		})
+
+		It("should discover regional cluster by AWS region", func() {
+			By("creating child ClusterDeployment without kof-regional-cluster-name label")
+			const childClusterDeploymentName = "test-child-aws"
+
+			const childClusterDeploymentConfig = `{
+				"region": "us-east-2",
+				"clusterLabels": {"k0rdent.mirantis.com/kof-cluster-role": "child"}
```

Comment:
Let's move it to a const, there are many places where this label is in use
```suggestion
				"clusterLabels": {kofClusterRoleLabel: "child"}
```